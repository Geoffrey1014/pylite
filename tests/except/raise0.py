def main():
    raise Exception() # throw instance

print "Start"
main()
print "End"
