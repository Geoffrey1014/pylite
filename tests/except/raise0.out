Start
Exception:
  At raise0.py, line 2, in main()
  At raise0.py, line 5, in __init__()
